pages/_app.tsx

```diff
@@ -31,7 +31,8 @@ const { chains, provider, webSocketProvider } = configureChains(
       ...bsc,
       iconUrl: bscIcon,
     },
-    // polygon,
+    polygon,
+    arbitrum,
```

Comment:
Arbitrum still isn't supported by railgun yet, right? Just tried using arbitrum and got a bunch of errors, but I think they'll fix themselves once we upgrade to an SDK version that supports it

Reply:
Yes, I checked their SDK and arbitrum is still commented out. So, I think they are running a little behind on adding support to the SDK because I think their target was the 24th.

Reply:
I will remove it from the wallet list